views/chat.ejs

```diff
@@ -13,7 +13,7 @@
             <% for(const message of messages) { %>
             <tr>
                 <td class="info"><%=  message.id %></td>
-                <td class="info"><%=  message.date %></td>
+                <td class="info"><%= new Date(message.date).toLocaleString("tr") %></td>
```

Comment:
The only language shouldn't be Turkish, we should support dynamic languages. I'll merge it, though.

Reply:
This is for best formatting. The Turkish formatting is very minimal. Not need add lang support. everybody can understand.
![image](https://user-images.githubusercontent.com/70021050/183292560-14724bc8-5578-48e5-a173-066e3ef8aed9.png)
But, we can delete "tr" string.
![image](https://user-images.githubusercontent.com/70021050/183292647-eb588f26-cd59-4c5e-ab27-91578bb9910c.png)